bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -90,26 +115,39 @@ impl WasmBezier {
 		self.0.length(None)
 	}
 
+	/// The wrapped return type is `Point`.
 	pub fn evaluate(&self, t: f64) -> JsValue {
-		vec_to_point(&self.0.evaluate(t))
+		let point: Point = vec_to_point(&self.0.evaluate(t));
+		to_js_value(point)
 	}
 
-	pub fn compute_lookup_table(&self, steps: i32) -> Vec<JsValue> {
-		self.0.compute_lookup_table(Some(steps)).iter().map(vec_to_point).collect()
+	/// The wrapped return type is `Vec<Point>`.
+	pub fn compute_lookup_table(&self, steps: i32) -> JsValue {
```

Comment:
`usize` not `i32`. And change the `compute_lookup_table` implementation to only use `usize` also.